live-examples/css-examples/basic-box-model/display.css

```
@@ -3,8 +3,12 @@
   height: 100%;
 }
 
+code {
+  background: #8888;
+}
+
 #example-element {
-  border: 3px solid #00f;
+  border: 3px solid orange;
```

```suggestion
  border: 3px dashed orange;
```
this enables using more than color as a reference in the discussion in the content.